attributes/default.rb

```diff
@@ -39,6 +47,9 @@
 when 'openbsd', 'freebsd', 'mac_os_x'
   default['ssl_certificate']['key_dir'] = '/etc/ssl'
   default['ssl_certificate']['cert_dir'] = '/etc/ssl'
+when 'windows'
+  default['ssl_certificate']['key_dir'] = Chef::Config[:file_cache_path]
+  default['ssl_certificate']['cert_dir'] = Chef::Config[:file_cache_path]
```

Comment:
Hmm I just checked and if you don't enable the CA role/feature, the `Certlog` folder doesn't exists, which'll be a bit painfull to create - special case for windows.

If you are OK I'ld prefer keep the `cache` folder, and any user can override the attribute to it's favorite place.